backend/api/Startup.cs

```diff
@@ -115,15 +118,15 @@ public void ConfigureServices(IServiceCollection services)
                 options.AddPolicy("Administrator", policy => policy.Requirements.Add(new RealmAccessRoleRequirement("administrator")));
             });
 
+            var cs = Configuration.GetConnectionString("PIMS");
```

Comment:
I wonder if the connection string is missing whether all this fails in a nice and polite way?

Reply:
I'll test this.

Reply:
Yes, it seems to handle that in a reasonable manner:

![image](https://user-images.githubusercontent.com/41091511/76561917-82f63400-6461-11ea-8dad-eaef33128103.png)